packages/api/src/constants.js

```diff
@@ -8,6 +8,7 @@ export const PIN_STATUSES = ['PinQueued', 'Pinning', 'Pinned', 'PinError']
 export const USER_TAGS = {
   ACCOUNT_RESTRICTION: 'HasAccountRestriction',
   DELETE_RESTRICTION: 'HasDeleteRestriction',
+  EARLY_ADOPTER: 'IsEarlyAdopter',
```

Comment:
Would exactly would this indicate?

Basically user records that were inserted before we shipped the first version of tiered pricing? or would it be something more nuanced/intentional for that? e.g. if a user was in early adopter status, but later chose to add a payment method and sign up for our most expensive subscription plan, would you expect that user to still have this tag?

another thought experiment. If we know we shipped tiered pricing on 2022-09-01, then I think we can just say 'all the users that were every early adopter are all the users that were inserted before 2022-09-01'.

What's an example of a user that wouldn't match that inserted_at query but that we would want to tag 'IsEarlyAdopter'?

Reply:
> if a user was in early adopter status, but later chose to add a payment method and sign up for our most expensive subscription plan, would you expect that user to still have this tag?

My initial thought was yes, but I'm totally on board with just using that user insertion date in order determine if someone is an early adopter.  It might have just been a miscommunication, but I think what you're suggesting would work.

Reply:
@e-schneid k. the only thing missing to use that heuristic would be the exact date to use as cuttover.

Reply:
I'll close the PR and address the date concerns in a later separate PR if needed.  I think our main interest is that we don't lose information about early adopters, and I think we've established that we're safe there since we'll always know the user insertion date.